assets/queries/openAPI/invalid_oauth2_token_url/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "3ba0cca1-b815-47bf-ac62-1e584eb64a05",
+  "queryName": "Invalid OAuth2 Token URL",
+  "severity": "MEDIUM",
+  "category": "Access Control",
+  "descriptionText": " OAuth2 security scheme flow requires a valid URL in the tokenUrl field",
+  "descriptionUrl": "https://swagger.io/specification/#security-requirement-object",
```

Comment:
I think https://swagger.io/specification/#oauth-flow-object makes more sense, since `tokenUrl` is field from OAuth Flow Object...